corehq/apps/userreports/templates/userreports/edit_report_config.html

```diff
@@ -36,8 +37,11 @@
   <div class="clearfix"></div>
 
   <ul class="nav nav-tabs">
-    <li class="active"><a data-toggle="tab" href="#tabs-configuration">Configuration</a></li>
-    <li><a data-toggle="tab" href="#tabs-usage">Usage</a></li>
+    <li class="active"><a data-toggle="tab" href="#tabs-configuration">{% trans "Configuration" %}</a></li>
+    <li><a data-toggle="tab" href="#tabs-usage">{% trans "Usage" %}</a></li>
+    {% if request|toggle_enabled:"LINKED_DOMAINS" %}
```

Comment:
Both upstream and downstream domains have this toggle enabled; should this be limited to downstream domains?

This might be a loophole in linked apps as well, it looks like you can make a chain of linked apps, not sure if we want to support that...I'll ping solutions about it.

Reply:
I'll leave as is for now, but you actually can't get to this page if the report is linked since the Edit button is hidden.